app/models/payment_notification/paypal.rb

```diff
@@ -20,7 +20,7 @@
 #  index_payment_notifications_on_type      (type)
 #
 
-class PaymentNotification::Paypal < PaymentNotification
+class PaymentNotification::PayPal < PaymentNotification
```

Comment:
This PR was only for the visual, didn't want to break it with the other changes.